Fixed some typos (#420)
valadas/Eraware_Dnn_Templates: README.md

https://marketplace.visualstudio.com/items?itemName=DanielValadas.eraware-dnn-templates

Currently this templates package only has a module template but it will probably grow into multiple different different types of DNN extensions.
In order for these templates to work, you will need Visual Studio 2019 (free edition is fine) and you must ensure you have the "Node.js development" workload installed, if you don't and you get an error opening the module.Web project, please use the visual studio installer to add it.

## Stecil SPA Module Template
A WebAPI/WebComponents module template.

> - The frontend uses [stencil.js](https://stenciljs.com/) to build standards compliant pure web components. The code enforces using [TypeScript](https://www.typescriptlang.org/) like Angular does and uses jsx like react does. JSX is not part of React but because it as been made popular together with React, you will see some React references that are only namespaces, this template does not use React. Stencil.js is a built-time only compiler that helps quickly and simply create stardards compliant WebComponents that can be used with any framework or without a framework. It also uses a subset of DNN specific WebComponents [dnn-elements](https://www.npmjs.com/package/@eraware/dnn-elements) that should make their way directly into DNN10 soon. Because TypeScript is enforced, a build will automatically document each component.

Reviewer: ```suggestion
- The frontend uses [stencil.js](https://stenciljs.com/) to build standards-compliant pure web components. The code enforces using [TypeScript](https://www.typescriptlang.org/) like Angular does and uses JSX like react does. JSX is not part of React but because it has been made popular together with React, you will see some React references that are only namespaces, this template does not use React. Stencil.js is a build-time-only compiler that helps quickly and simply create standards-compliant WebComponents that can be used with any framework or without a framework. It also uses a subset of DNN specific WebComponents [dnn-elements](https://www.npmjs.com/package/@eraware/dnn-elements) that should make their way directly into DNN10 soon. Because TypeScript is enforced, a build will automatically document each component.
```

Reviewer 2: Awesome 🌮
